shardingsphere-governance/shardingsphere-governance-repository/shardingsphere-governance-repository-provider/shardingsphere-governance-repository-zookeeper-curator/src/main/java/org/apache/shardingsphere/governance/repository/zookeeper/CuratorZookeeperRepository.java

```diff
@@ -262,13 +264,13 @@ private void addCacheData(final String cachePath) {
         caches.put(cachePath + PATH_SEPARATOR, cache);
     }
     
-    private Type getChangedType(final CuratorCacheListener.Type type) {
+    private Type getChangedType(final TreeCacheEvent.Type type) {
```

Comment:
why not use a map here ?

Reply:
I'm not sure that is there any difference for a few items in switch?